lib/ex_rlp/encode.ex

```diff
@@ -23,7 +23,7 @@ defimpl ExRLP.Encode, for: BitString do
     <<prefix>> <> item
   end
 
-  defp encode_item(item) when is_binary(item) do
+  defp encode_item(item) do
```

Comment:
why did you remove guards? What is the benefit?

Reply:
@ayrat555 the benefit is performance. Second, it doesn’t make sense to check if the argument is binary in you’re implementing the binary protocol, same applies for integer protocol. It’s defensive.